### DISCUSSION TITLE
Hide "none" in forced combinaisons

### DISCUSSION THREAD

**Discussion by BedoSensei** [nftchef/art-engine#280] [Help + Q&A]
Hello,

The function to hide in the metadatas the "None" layers works perfectly but not for the forced combinations.

I need the None layer for some forced configurations but I don't want this layer to appear in the metadatas.
Is there a way to hide None layers from forced combinations ?

Thanks in advance.

**Comment by nftchef** [comment]
`None` in forcedCombinations is a bit problematic because it will force NONE for every folder/layer it is found in.

when an entire layer shows up in the metadata, the best tool is to use the `removeTrait` util after generation https://github.com/nftchef/art-engine#remove-trait-util
